en/docs/develop/scim2-batch-operations.md

```diff
@@ -0,0 +1,1178 @@
+           </td>
+       </tr>
+       <tr>
+           <td><code>display</code></td>
+           <td>Required</td>
+           <td>
+               The display name of a user assigned to the group.</br>
+               <b>Possible values:</b> The username.
+           </td>
+       </tr>
+       <tr>
+           <td><code>value</code></td>
+           <td>Required</td>
+           <td>
+               The ID of the user.</br>
+               <b>Possible values:</b> The user ID.
+           </td>
+       </tr>
+    </table>
+ 
+### Delete users
+ 
+Given below is an example request payload to delete existing user groups in bulk. This request includes an array of operations that delete multiple user groups.
+ 
+```json
+{
+   "schemas":["urn:ietf:params:scim:api:messages:2.0:BulkRequest"],
+   "Operations":[
+       {
+           "method": "DELETE",
+           "path": "/Groups/46887262-2ba1-4cee-b3ef-64549423e0b0" 
+       },
+       {
+           "method": "DELETE",
+           "path": "/Groups/18d04a36-0894-4f71-bdc4-2610fcc6ae42"
+       }
+   ]
+}
+```
+ 
+The parameters in the request body are explained below.
+ 
+-   Main parameters
+ 
+    <table>
+       <tr>
+           <th>Parameter</th>
+           <th>Required/Optional</th>
+           <th>Description</th>
+       </tr>
+       <tr>
+           <td><code>failOnErrors</code></td>
+           <td>Optional</td>
+           <td>The number of errors that will be accepted by WSO2 IS before returning the response.</br>
+               <b>Possible values</b>: An integer.
+           </td>
+       </tr>
+       <tr>
+           <td><code>schemas</code></td>
+           <td>Required</td>
+           <td>This is the schema that is required for sending bulk requests:</br>
+           <code>urn:ietf:params:scim:api:messages:2.0:BulkRequest</code>.
+           </td>
+       </tr>
+       <tr>
+           <td><code>operations</code></td>
+           <td>Required</td>
+           <td>Array of  operations. To delete multiple user groups, add an array of PUT operations. You can include any number of operations in one bulk request.</br>
+           The parameters that can be used for the operation are explained below.
+           </td>
+       </tr>
+    </table>
+ 
+-   Operation parameters
+ 
+    <table>
+       <tr>
+           <th>Parameter</th>
+           <th>Required/Optional</th>
+           <th>Description</th>
+       </tr>
+       <tr>
+           <td><code>method</code></td>
+           <td>Required</td>
+           <td>
+               The method that should be used in the operation.</br>
+               <b>Possible value</b>:<code>DELETE</code>.
+           </td>
+       </tr>
+       <tr>
+           <td><code>path</code></td>
+           <td>Required</td>
+           <td>
+               Add this path to specify the existing user group that should be deleted.</br>
+               <b>Possible value</b>:<code>/Groups/{group_id}</code>.
+           </td>
+       </tr>
+    </table>
```

Comment:
Need to mention 
NOTE: Even though the multiple resources in the same resource type are managed per request in the above examples, it is allowed to do different operations (POST, PATCH, PUT, DELETE) on different resources (Users, Groups, …) in a single POST /Bulk operation.

Reply:
Added this note in two places:

```
The examples given below show individual resource operations (POST, PATCH, PUT, or DELETE) handled in a single request. However, note that a single request can execute a combination of operation types simultaneously.
```